dashboard/src/repositories/postgres/bugReports.ts

```diff
@@ -0,0 +1,26 @@
+import prisma from '@/lib/postgres';
+import {
+  type BugReport,
+  type BugReportCreate,
+  BugReportCreateSchema,
+  BugReportSchema,
+} from '@/entities/bugReport';
+
+export async function createBugReport(data: BugReportCreate): Promise<BugReport> {
+  try {
+    const payload = BugReportCreateSchema.parse(data);
+
+    const report = await prisma.bugReport.create({
+      data: {
+        userId: payload.userId,
+        dashboardId: payload.dashboardId ?? null,
```

Comment:
Since this one comes from the auth context, is there ever a case where it can be nullish? 😄